pages/learn/glossary.md

```diff
@@ -124,6 +124,12 @@ Said more plainly, whether a particular value is a subschema or not depends on i
 
 Subschemas may themselves contain sub-subschemas, though colloquially one generally uses the term "subschema" regardless of the level of nesting, further clarifying which larger schema is the parent schema whenever needed.
 
+
+### anchor 
+
+The `$anchor` keyword is one of the ways to identify the location of a subschema within a document. By applying an anchor, the subschema becomes identifiable via a plain-name URI string. An anchor is also a shorter alternative to using a JSON Pointer for identifying a subschema. 
```

Comment:
I'd at least flip the order of the sentences here -- we're defining `anchor` (the concept) not `$anchor` (the keyword which helps create them).

Also I wouldn't assume someone reading this knows what a "plain-name URI string" is -- levelwise, I'd assume the person reading this barely knows what a *URI* itself is!  Giving a one sentence example of what that is is likely useful.

Also cross-link to `JSON Pointer` if we have an entry for that (and if not add a TODO probably).

Reply:
We have pointer coming in another PR that's not merged yet.

I also agree with the above.

Reply:
@Julian Thank you for the comments. 

I redefined the `anchor` term and added a URI example to my new definition. @gregsdennis I added a TODO note to add a "see more" to the JSON pointer when the other PR is merged.